GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/ClassificationPanelViewModel.cs

```diff
@@ -356,8 +376,16 @@ public void DropSubject(TableSubject subject)
             GlobalData.GetInstance().Logger?.AddEntry("Drop_Galaxy", User.Name, subject.Id, CurrentView);
             if (CheckAlreadyCompleted(subject)) return;
             if (CurrentView == ClassifierViewEnum.SummaryView) CurrentView = ClassifierViewEnum.SubjectView;
-            LoadSubject(subject);
-            NotifySpaceView(RingNotifierStatus.IsCreating);
+            if (_localDBService.CheckIfSubjectRetired(subject.Id))
```

Comment:
If you change this line to `if (true)` in your local code then you should see the retired modal appear every time you drag a subject into the classifier. Unfortunately, the current subject set doesn't have many retired subjects hitting the 25 classification threshold.